.gitignore

```
@@ -9,6 +9,6 @@ jest-preset.json
 # misc
 .DS_Store
 npm-debug.log
-
+*.tgz
```

probably better to keep the newline between sections convention seen in the other sections
```suggestion
*.tgz

```

---

Taken care of in 008ecd9